src/utils/create_statistics.py

```diff
@@ -0,0 +1,104 @@
+import json
+import re
+
+
+#TODO: rename Bogen to Sheet
+class Box:
```

Comment:
That is true, but since we write the object to json, use it for the NN and again write in to json **in another format**, I have just defined another Box class. If you'd like I can rename it. It is only used locally though.